pywemo/ssdp.py

```diff
@@ -213,7 +243,7 @@ def build_ssdp_request(ssdp_st, ssdp_mx):
             'ST: {}'.format(ssdp_st),
             'MX: {:d}'.format(ssdp_mx),
             'MAN: "ssdp:discover"',
-            'HOST: 239.255.255.250:1900',
+            'HOST: {}:{}'.format(MULTICAST_GROUP, MULTICAST_PORT),
```

Comment:
I think f-strings would be cleaner here - as well as the couple unchanged lines above.  They probably used format before mainly due to python 2.7 support previously.

Reply:
I switched the code around to use f-strings.